docs/install/setting_up_cluster_on_amazon_web_services.md

```diff
@@ -208,31 +205,35 @@ Here:
 
 * `<your-keyfile-path>` refers to the path to the key file.
 
-* `<action>` refers to the action to be performed. You must first launch your cluster using the `launch` action. The `start` and `stop` actions can be used to manage the nodes in the cluster.
+* `<action>` refers to the action to be performed. Some of the available actions are `launch`, `destroy`, `stop`, `start` and `reboot-cluster`.
+Use `launch` action to create a new cluster while `stop` and `start` actions work on existing clusters.
 
 By default, the script starts one instance of a locator, lead, and server each.
-The script identifies each cluster by its unique cluster name (you provided) and internally ties members (locators, leads, and stores/servers) of the cluster with EC2 security groups.
+The script identifies each cluster by its unique cluster name (that you provide) and internally ties members (locators, leads, and stores/servers) of the cluster with EC2 security groups - whose names are also derived from the cluster name.
 
-The  names and details of the members are automatically derived from the provided cluster name. When running the script you can also specify properties like the number of stores and region.
+When running the script you can also specify options like the number of stores in the cluster and region where the EC2 instances should be launched.
```

Comment:
When running the script, you can also specify options such as the number of stores in the cluster and the region where the EC2 instances should be launched.